sixtyfps_compiler/passes/materialize_fake_properties.rs

```diff
@@ -77,3 +82,34 @@ fn has_declared_property(elem: &Element, prop: &str) -> bool {
         _ => false,
     }
 }
+
+/// Initialize a sensible default binding for the now materialized property
+fn initialize(elem: ElementRc, name: &str) {
+    let expr = match name {
+        "minimum_height" => layout_constraint_prop(&elem, "min_height"),
```

Comment:
Perhaps either these two functions or just the default binding definitions could go into the `typeregister.rs` or somewhere else, just so that the definition of our reserved properties and associated details like defaults are in one place. This doesn't necessarily belong into this change set though.